components/dashboard/src/projects/Project.tsx

```diff
@@ -362,6 +370,13 @@ export default function () {
                     </>
                 )}
             </div>
+            <div>
+                {isLoading && (
+                    <div className="flex justify-center">
+                        <img alt="" className="h-4 w-4 animate-spin" src={Spinner} />
+                    </div>
+                )}
+            </div>
```

Comment:
Yes that would be ideal, unfortunately the spinner is necessary here because it takes a while until we get a response back to know which prebuild logs page to go to..

Reply:
Alternatively, we could _not_ attempt to redirect until we have a page we can render in time.

Reply:
> Alternatively, we could _not_ attempt to redirect until we have a page we can render in time.

Hmm, how would the UX be like here? Also looping in @jldec

Reply:
Showing a spinner near the action that the user clicked, and then redirecting after a pause, is still preferable over not redirecting at all.